client/lib/analytics/test/google-analytics-4.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @jest-environment jsdom
+ */
+import isJetpackCloud from 'calypso/lib/jetpack/is-jetpack-cloud';
+import { TRACKING_IDS } from '../ad-tracking/constants';
+import { setup, firePageView, Ga4PropertyGtag } from '../ad-tracking/google-analytics-4';
+
+jest.mock( 'calypso/lib/jetpack/is-jetpack-cloud' );
+
+describe( 'Google Analytics 4 implementation', () => {
+	beforeEach( () => {
+		jest.spyOn( window, 'gtag' ).mockImplementation( () => [] );
+		// window.gtag = jest.fn( () => [] );
```

Comment:
Unused comment?

Reply:
I was testing that approach first, but realized the new approach was better. Removing it now.